src/components/filters/filters.jsx

```diff
@@ -1,50 +1,73 @@
-        <>
-        <aside className="filters-wrapper  flex-column p1">
-            <form className="drawer flex-column" action="">
-                <div className="heading h2">
-                    Filters
-                </div>
-                <label className=" h3 m1" for="">Select Price</label>  
-                
-                    <div>
-                        <label className="p1 h5"> Price Above ₹ 20,000 </label>
-                        <input type="checkbox" />
-                    </div>
-                    <div>
-                        <label className="p1 h5">Price Above ₹ 30,000 </label>
-                        <input type="checkbox" />
-                    </div>
-                
-                <label className="m1 h3">Ratings</label>
-                <div>
-                    <label className="p1 h5">5 Stars and above</label>
-                    <input type="radio" value="Running" />
-                </div>
-                <div>
-                    <label className="p1 h5">4 Stars and above</label>
-                    <input type="radio" value="Running" />
-                </div>
-                <div>
-                    <label className="p1 h5">3 Stars and above</label>
-                    <input type="radio" value="Running" />
-                </div>
-                <label className="m1 h3">Sort by</label>
-                <div>
-                    <label className="p1 h5">Price Low to High</label>
-                    <input type="radio" value="Running" />
-                </div>
-                <div>
-                    <label className="p1 h5">Price High to Low</label>
-                    <input type="radio" value="Running" />
-                </div>
-            </form>
-        </aside>
-        </>
-    )
-}
+const Filters = () => {
+  const { filterDispatch, filterState: {  priceRange } } = useCart()
+    
+  return (
+    <>
+      <aside className="filters-wrapper  flex-column p1">
+        <form className="drawer flex-column" action="">
+          <div className="heading h2">Filters</div>
+          <label className=" h3 m1" for="">
+            Select Category
+          </label>
 
-export {Filters}
\ No newline at end of file
+          <div>
+            <label className="p1 h5"> Running </label>
+            <input type="checkbox" name="category"  onChange={()=> filterDispatch({type:"CATEGORY" , payload:"running"})} />
+          </div>
+          <div>
+            <label className="p1 h5"> Sneakers </label>
+            <input type="checkbox" name="category"  onChange={()=> filterDispatch({type:"CATEGORY" , payload:"sneakers"})}/>
+          </div>
+          <label className=" h3 m1" for="">
+            Select Price
+          </label>
+          <input value={priceRange} type="range" min={2500} max={12000} 
+          onChange={(e)=> filterDispatch({type:"PRICE_RANGE" , payload:e.target.value})} />
+          
+          <label className=" h3 m1" for=""> ₹ {priceRange}</label>
```

Comment:
Use `htmlFor` instead of 'for' here. React ignores 'for' attribute of the label element. Since 'for' is a reserved word in JavaScript, React elements use htmlFor instead.

Reply:
Thank you for the suggestion, will implement this.